template/desktops/xfce.sh

```diff
@@ -19,6 +19,39 @@ xfconf-query -c xfce4-session -p /startup/ssh-agent/enabled -n -t bool -s false
 xfconf-query -c xfce4-session -p /startup/gpg-agent/enabled -n -t bool -s false
 echo "TIMING: $(date -Iseconds) - Disabled startup services"
 
+# Turn off power saving measures that turn off the display
+xfconf-query \
+    --channel xfce4-power-manager \
+    --property /xfce4-power-manager/dpms-enabled \
+    --create \
+    --type bool \
+    --set true
+
+xfconf-query \
+    --channel xfce4-power-manager \
+    --property /xfce4-power-manager/dpms-on-ac-off \
+    --create \
+    --set 0 \
+    --type uint
+
+xfconf-query \
+    --channel xfce4-power-manager \
+    --property /xfce4-power-manager/dpms-on-ac-sleep \
+    --create \
+    --set 0 \
+    --type uint
+
+xfconf-query \
+    --channel xfce4-power-manager \
+    --property /xfce4-power-manager/blank-on-ac \
```

Comment:
Confirmed this property is `int` type.

https://gitlab.xfce.org/xfce/xfce4-power-manager/-/blob/xfce4-power-manager-1.6.6/src/xfpm-power.c#L1155

```c
    g_object_class_install_property (object_class,
                                     PROP_ON_AC_BLANK,
                                     g_param_spec_int  (ON_AC_BLANK,
                                                        NULL, NULL,
                                                        0,
                                                        G_MAXINT16,
                                                        10,
                                                        XFPM_PARAM_FLAGS));
```